mongodbatlas/resource_mongodbatlas_cluster.go

```diff
@@ -907,6 +920,17 @@ func resourceMongoDBAtlasClusterUpdate(ctx context.Context, d *schema.ResourceDa
 		}
 	}
 
+	if d.Get("paused").(bool) {
```

Comment:
this can never be nil right? becase it has a default value.

Reply:
That's correct, tony